morph_tool/transform.py

```diff
@@ -225,7 +233,7 @@ def align_morphology(
         L.info('We did not find an apical point to align the morphology')
         return np.eye(3)
 
-    principal_direction = _get_principal_direction(points)
+    principal_direction = _get_principal_direction(points-np.linalg.norm(morph.soma.points, axis=0))
```

Comment:
Shouldn't this be `points - np.mean(morph.soma.points, axis=0)` ?

Reply:
Also, keep in mind that the rotation has to be done around the same origin (center of soma).

Reply:
something like that?